data/apply-planet_osm_line.sql

```diff
@@ -33,6 +33,8 @@ CREATE INDEX planet_osm_line_transit_geom_15_index ON planet_osm_line USING gist
 
 CREATE INDEX planet_osm_line_waterway_geom_index ON planet_osm_line USING gist(way) WHERE waterway IN ('canal', 'river', 'stream', 'dam', 'ditch', 'drain');
 
+CREATE INDEX planet_osm_line_piste_geom_index ON planet_osm_line USING gist(way) WHERE tags ? 'piste:type';
```

Comment:
I did that initially, and then worried that the list of `piste:type` that I pulled from the top 10 on [taginfo](http://taginfo.openstreetmap.org/keys/piste:type#values) was incomplete. Checking on taginfo, it seems like the top 10 encompasses 95% or more of the data, so including all those values wouldn't bloat the index too much.

Do you think it should be more precise?